src/plugins/analysis/binwalk/code/binwalk.py

```diff
@@ -1,13 +1,12 @@
-import os
 import string
+from pathlib import Path
 from tempfile import TemporaryDirectory
+from typing import List
 
-from common_helper_files import get_binary_from_file
```

Comment:
Note: We should find all occurrences of this and replace them in a separate branch by pathlib, as you did here

Reply:
I concur!